app/controllers/claims_controller.rb

```diff
@@ -89,6 +88,8 @@ def redirect_to_existing_claim_journey
   end
 
   def set_backlink_path
+    return if navigator.current_slug == "confirmation"
```

Comment:
Might be one to throw in `DEAD_END_SLUGS` so we keep all the show / don't show back link configuration in one place